app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    splitted = command.split()
```

Comment:
You must check if splitted has three arguments because if not, the code will fail.

Reply:
command_name, filename, copy_filename = command.split() and you can do like this